src/vm/stackwalk.h

```diff
@@ -45,6 +45,10 @@ class AppDomain;
  #endif 
 #endif // _TARGET_X86_ && !FEATURE_PAL
 
+#if !defined(_TARGET_X86_)
+#define RECORD_RESUMABLE_FRAME_SP
```

Comment:
I tried, and there is no notable difference. If this code is related with Funclet-based EH, I will revise this line as as another PR  (I would like not to make behavior change here).